generic-ec-curves/Cargo.toml

```diff
@@ -9,18 +9,25 @@ repository = "https://github.com/dfns/generic-ec"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-generic-ec-core = "0.1"
+generic-ec-core = { version = "0.1", path = "../generic-ec-core", default-features = false }
 
 subtle = { version = "2.4", default-features = false }
 rand_core = { version = "0.6", default-features = false }
 zeroize = { version = "1", default-features = false }
 
-crypto-bigint = { version = "0.5", optional = true }
-elliptic-curve = { version = "0.13", features = ["sec1", "hash2curve"], optional = true }
-k256 = { version = "0.13", optional = true, features = ["hash2curve"] }
-p256 = { version = "0.13", optional = true, features = ["hash2curve"] }
-sha2 = { version = "0.10", optional = true }
-stark-curve = { version = "0.1", optional = true }
+crypto-bigint = { version = "0.5", default-features = false, optional = true }
+elliptic-curve = { version = "0.13", default-features = false, features = [
+    "sec1",
+    "hash2curve",
+], optional = true }
+k256 = { version = "0.13", optional = true, default-features = false, features = [
+    "hash2curve",
+] }
+p256 = { version = "0.13", optional = true, default-features = false, features = [
+    "hash2curve",
+] }
```

Comment:
Style nitpick. Was this an autoformatter? I much preferred the one-line style

Reply:
Yup, vscode just messed the file, let me fix that.

Reply:
Me too. If it's too long, I'd prefer

```toml
[dependencies.elliptic-curve]
version = "0.13"
default-features = false
features = ["sec1", "hash2curve"]
optional = true
```